azurerm/resource_arm_subnet.go

```diff
@@ -125,6 +125,11 @@ func resourceArmSubnet() *schema.Resource {
 					},
 				},
 			},
+
+			"private_link_service_network_policies": {
```

Comment:
Does this want to be
```suggestion
			"enable_private_link_service_network_policies": {
```

Reply:
Actually, to match AZ CLI I changed this to `disable_private_link_service_network_policies`